### PR TITLE
Python bindings - Fixes issue #471

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

File: native_client/Makefile [hunk 1]
```diff
@@ -38,10 +41,30 @@ endif
...
 
 clean:
-	rm -f deepspeech
+	rm -rf build dist deepspeech.egg-info
+	rm -f deepspeech setup.cfg python/deepspeech_wrap.cpp python/deepspeech.py
 
 deepspeech: client.cc
 	$(TOOLCHAIN)$(CXX) -o deepspeech $(CFLAGS) client.cc $(LDFLAGS)
 
+setup.cfg: setup.cfg.in
```
Comment (lissyx) [inline native_client/Makefile]: Nit: I don't see changes to .gitignore, is `setup.cfg.in` properly ignored ?

**Comment by Cwiiis** [comment]
Just added all the generated files to the .gitignore, good call :)

**Comment by kdavis-mozilla** [comment]
@Cwiiis Did you look into the build failures on Taskcluster? It looks like C++11 is not being used to compile, for example `nullptr` is not found.

**Comment by Cwiiis** [comment]
I haven't looked at the build failures, I can't seem to figure out how to view the logs on Taskcluster :/ I expect I just need to add some flag in the build file.

**Comment by Cwiiis** [comment]
Just pushed with a flag to enable C++11 when building libdeepspeech. Fingers crossed!

**Comment by Cwiiis** [comment]
Realised I used a flag that's only supported in 4.7+, pushed again with the flag that's supported in 4.6.

**Comment by Cwiiis** [comment]
ok, must be pretty old gcc, just switching to NULL instead of nullptr.

**Pull request merged**